### ISSUE TITLE
Add timestamp to the log in Output Window

### ISSUE THREAD

**Issue by suimong** [shanalikhan/code-settings-sync#1064]
I'd like to suggest that a timestamp being added to the sync log ("CODE SETTINGS SYNC UPLOAD SUMMARY") shown in the Output Window. It's a useful piece of information and presumably not hard to be implemented. Thanks!

**Comment by shanalikhan** [comment]
Nice suggestion. Will be done in next version.

**Comment by bernardoadc** [comment]
Yes its simple but has great value. Sometimes I look to my window and there's all that nice output, but I have no idea when that work was done.

**Comment by n4nn31355** [comment]
Just looking through the issues. It seems to be already done in dev branch: https://github.com/shanalikhan/code-settings-sync/commit/066a4d769141ed2ce2ed5ed5296ef7419a4f488f

**Comment by shanalikhan** [comment]
@n4nn31355 I received your message in slack, lets discuss if you are available.
